industrial_ci/src/builders/colcon.sh

```diff
@@ -40,7 +40,13 @@ function builder_run_tests {
     else
         output_handler="console_cohesion+"
     fi
-    ici_exec_in_workspace "$extend" "$ws" colcon test --event-handlers "${_colcon_event_handlers[@]}" "${output_handler}"
+    local opts=(--event-handlers "${_colcon_event_handlers[@]}" "${output_handler}")
+    if [ "$PARALLEL_TESTS" == false ]; then
+        opts+=("--executor sequential")
+    else
+        opts+=("--executor parallel")
```

Comment:
```suggestion
        opts+=(--executor parallel)
```

Reply:
why do we need to remove the quotes?
should this be done for all occurrences, i.e.
 - https://github.com/ros-industrial/industrial_ci/pull/446/files#diff-46bbdbaf63f42dea8f37320cc6574e1aR33
 - https://github.com/ros-industrial/industrial_ci/pull/446/files#diff-9fbeddd0a763e900f5b2109a24aceb2aR36
 - https://github.com/ros-industrial/industrial_ci/pull/446/files#diff-807dad9df09c471416c53ddfe3af7795R38
 - https://github.com/ros-industrial/industrial_ci/pull/446/files#diff-807dad9df09c471416c53ddfe3af7795R41
 - https://github.com/ros-industrial/industrial_ci/pull/446/files#diff-807dad9df09c471416c53ddfe3af7795R44
 - https://github.com/ros-industrial/industrial_ci/pull/446/files#diff-60272b6836288cfae2c2fabe8971649fR45
 - https://github.com/ros-industrial/industrial_ci/pull/446/files#diff-60272b6836288cfae2c2fabe8971649fR47

Reply:
> why do we need to remove the quotes?

because with quote it one string `"--executor parallel"`, without it is `"--executor"``and `"parallel"`, which is needed for proper parsing.

> should this be done for all occurrences, i.e.

Single strings are fine. I will have another look.